smarts/core/remote_agent_buffer.py

```diff
@@ -40,7 +40,8 @@ def __del__(self):
         self.destroy()
 
     def destroy(self):
-        atexit.unregister(self.destroy)
+        if atexit.unregister is not None:
```

Comment:
This is odd, why do we need it? Isn't this is always true?

Reply:
It does not seem to always be true. I am not sure why.

Reply:
Pretty strange, docs don't indicate unregister could be none. Do you have an example snippet in the logs?

Reply:
I was seeing this in the logs:

![image](https://user-images.githubusercontent.com/8764693/99978991-9f7a9800-2d74-11eb-8065-2285de1d6b98.png)